manager: replace stod by boost spirit parsing

rtrepos opened the pull request:
Using cvle, the reading of double from char* can lead to error when value is very low. (as mentionned here https://stackoverflow.com/questions/48086830/stdstod-throws-out-of-range-error-for-a-string-that-should-be-valid) We prefer using the boost spirit parsing as in the SaxParser code.

quesnel commented:
You can use the traditional C function `strtod` (in cstdlib [1]) to convert string to double without exception and error. I rewrite the example: [2].

[1] https://en.cppreference.com/w/c/string/byte/strtof
[2] https://www.godbolt.org/z/7W44xbq3h

rtrepos commented:
Ok thank you goth.
It worked by solving my issue. 
But the idea also was to be sure to have the same behavior when with use the parallelization or not. In one case we would rely on boost spirit, on the other on the strtod. Do you think it is the case ?

quesnel commented:
Le jeu. 22 août 24, rtrepos a écrit :
> But the idea also was to be sure to have the same behavior when with
> use the parallelization or not. In one case we would rely on boost
> spirit, on the other on the strtod. Do you think it is the case ?

Usually yes. The C standard ensures the behaviour of this function.

rtrepos committed:
manager: replace stod by strtod

 Using cvle, the reading of double from char* can lead to error when value is very low.
 (as mentionned here https://stackoverflow.com/questions/48086830/stdstod-throws-out-of-range-error-for-a-string-that-should-be-valid)
 This error does not occur with the C function strtod.

rtrepos commented:
Ok thank you, I stoped the CI/CD checks (that were still waiting for ressources) and I close this pull request and I push new commits on master and master2.1.
